### PR TITLE
Ohorodniy Boris 203

### PR DESCRIPTION
**Pull request by DadyaBorya**

### CODE REVIEW

File: app/css/app.css [hunk 1]
```diff
@@ -25,11 +26,21 @@ footer, header, hgroup, menu, nav, section {
   display: block;
 }
```
Comment (Zadvornyi) [inline app/css/app.css:28]: Need to fix:

follow BEM folder structure
use & in scss files
follow BEM naming recommendation
https://scalablecss.com/bem-nesting-grandchild-elements/

P.S for every block create folder

File: app/index.html [hunk 2]
```diff
@@ -8,7 +8,96 @@
...
 <body>
 
-<script src="js/bundle.js"></script>
+<header class="header">
+  <div class="header__logo">
+    <a href="/" class="header__logo-link">
+      <img src="./img/github-mark-white.svg" width="32" height="32" alt="Логотип">
+    </a>
+  </div>
+  <nav class="header__nav">
```
Comment (Zadvornyi) [inline app/index.html:17]: create BEM folder structure for this 'header__nav'

File: app/index.html [hunk 3]
```diff
@@ -8,7 +8,96 @@
...
+  <div class="main__content">
+    <div class="main-title">
+      Built for developers
+    </div>
+    <div class="main__content-subtitle">
+      GitHub is a development platform inspired by the way you work. From <span>open source</span> to <span>business</span>, you can host and review code, manage projects, and build software alongside 50 million developers.
+    </div>
+  </div>
+
+  <form class="login-form__content">
```
Comment (Zadvornyi) [inline app/index.html]: form must have attr name

Comment (Zadvornyi) [reply app/index.html]: Follow namin convantion for Block => element +=> modificator

File: app/scss/app.scss [hunk 4]
```diff
@@ -1,3 +1,149 @@
...
+body {
+  background-color: $color-gray;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $color-dark-gray;
+  padding: 16px 4px;
```
Comment (Zadvornyi) [inline app/scss/app.scss]: Красава зроби так для інших стилів

**Comment by DadyaBorya** [comment]
Done !

**Pull request closed without merging**